components/AddButton/index.js

```diff
@@ -5,7 +5,7 @@ import styled from "styled-components";
 export default function AddButton({ handleClick }) {
   return (
     <AddProjectButton onClick={handleClick}>
-      <FontAwesomeIcon icon={faCirclePlus} />
+      <FontAwesomeIcon icon={faCirclePlus} style={{ float: "left" }} />
     </AddProjectButton>
   );
 }
```

Comment:
Since we want to style our applications with styled components, you should avoid inline styling, even if it's just one line of style.